server/render/index.js

```diff
@@ -68,12 +67,18 @@ export function render( element, key = JSON.stringify( element ) ) {
 	//todo: render an error?
 }
 
-export function serverRender( context ) {
-	if ( config.isEnabled( 'server-side-rendering' ) ) {
+export function serverRender( req, res ) {
```

Comment:
Scratch that, I was feverishly misreading the diff. So [this block](https://github.com/Automattic/wp-calypso/pull/4327/files#diff-d92d930a21711a960b300aeb231d9056R79) is outside the `config.isEnabled()` conditional, and will hence always render either `index.jade` or `desktop.jade`.